### ISSUE TITLE
Changing the module package source should reset the search form

### ISSUE THREAD

**Issue by DataTraveler1** [ironmansoftware/powershell-universal#2412]
### Summary of the new feature / enhancement

If a search in the 'local' modules is conducted and then the pull-down for package source is changed to PSGallery, the same search occurs again. It could be ideal if the search form were to be reset when the pull-down is changed to a new package source.

![issue2412](https://github.com/ironmansoftware/issues/assets/537287/a162cbbc-a7c5-4345-830e-6b084538ab0e)

In other words, after I selected 'PSGallery', I thought the previous search would be cleared and I would start a new search.
